test/pasta-curves-tests.t.sol

```diff
@@ -185,7 +176,7 @@ contract PastaCurvesContractTests is Test {
 
         uint256 scalar = 0x0716ea7ecbe554f483e62dc4a31ef8d080b35ea748906aa770491c144257819a;
 
-        Pallas.PallasProjectivePoint memory p1MulScalar = pallas.projectiveScalarMul(p1, scalar);
+        Pallas.PallasProjectivePoint memory p1MulScalar = Pallas.projectiveScalarMul(p1, scalar);
```

Comment:
I guess, here it should be `Pallas.scalarMul`, as soon as we call library's function directly

Reply:
Yes, but that code is commented out. The line change was due to a "replace all" command